polaris-tokens/scripts/toJSON.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+
+const {tokens} = require('../dist/index');
+
+function getFileName(fileName) {
+  return path.join(process.cwd(), 'dist', fileName);
```

Comment:
```suggestion
  return path.join(__dirname, '../dist', fileName);
```

`process.cwd()` is relative to where the script is executed and thus will have different results if ever run from the root of the monorepo.